File: core/src/main/java/com/jigumulmi/admin/banner/dto/response/CreateBannerResponseDto.java
package com.jigumulmi.admin.banner.dto.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonProperty.Access;
import com.jigumulmi.admin.banner.dto.AdminCreateBannerImageS3KeyDto;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class CreateBannerResponseDto {

    private Long bannerId;

    @Schema(hidden = true)
    @JsonProperty(access = Access.WRITE_ONLY)
    private AdminCreateBannerImageS3KeyDto s3KeyDto;
}

File: admin/src/main/kotlin/com/jigumulmi/banner/dto/response/CreateBannerResponseDto.kt
package com.jigumulmi.banner.dto.response

import com.fasterxml.jackson.annotation.JsonProperty
import com.jigumulmi.banner.dto.AdminCreateBannerImageS3KeyDto
import io.swagger.v3.oas.annotations.media.Schema

data class CreateBannerResponseDto(
    val bannerId: Long,

    @Schema(hidden = true)
    @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
    val s3KeyDto: AdminCreateBannerImageS3KeyDto
)
